.travis.yml

```diff
@@ -4,6 +4,8 @@ rvm:
   - 2.2.3
 install:
   - bundle install
+  - nvm install 5.0
+  - nvm use 5.0
   - npm install -g npm
```

Comment:
@justin808 the main build failure is from a c++ compiler error for `NAN` module used in sleep. I will fix this.